unittests/resources/test.sxfm

```diff
@@ -0,0 +1,53 @@
+<feature_model name="apache">
+<feature_tree>
+:r root(root)
+	:g [1,1] threadCount(threadCount)
+		: threadCount_256(threadCount_256)
+		: threadCount_128(threadCount_128)
+		: threadCount_64(threadCount_64)
+		: threadCount_32(threadCount_32)
+		: threadCount_16(threadCount_16)
+	:g [1,1] alt1(encryption)
+        : a1 (none)
+        :g [1,1] alt3(asymmetric)
+            : a2 (ECC)
+            : a3 (RSA)
+        :g [1,1] alt4(symmetric)
+            : a4 (AES)
+            : a4 (DES)
+	:g [1,1] compressionLevel(compressionLevel)
+		: compressionLevel_9(compressionLevel_9)
+		: compressionLevel_5(compressionLevel_5)
+		: compressionLevel_1(compressionLevel_1)
+	:g [1,1] processCount(processCount)
+		: processCount_64(processCount_64)
+		: processCount_32(processCount_32)
+		: processCount_16(processCount_16)
+		: processCount_8(processCount_8)
+	:g [1,1] mpm(mpm)
+		: worker(worker)
+		: event(event)
+	:o tls(tls)		:o aesTls(aesTls)			:g [0,1] Group_0(Group_0)
```

Comment:
are multiple nodes in one line allowed?

Reply:
Nope. Seems to be a copy & paste error. This is fixed in the next revision. Thanks ;-)